vmdb/app/models/configured_system_foreman.rb

```diff
@@ -0,0 +1,4 @@
+class ConfiguredSystemForeman < ConfiguredSystem
+  belongs_to :operating_system_flavor
+  has_and_belongs_to_many :customization_scripts
```

Comment:
This is interesting...It wasn't in the drive-by design.  Is it needed?  Where would this information be used.  My only concern is that if it's for genealogy purposes it would get stale.  That is, if a script is changed, then this would point to the "new" script, which would make the relationship invalid or inaccurate.  You would have to keep a whole history of scripts, which can become a huge maintenance hassle.